### ISSUE TITLE
Importing SVG results in import.meta.url appearing in final build file

### ISSUE THREAD

**Issue by lucaseverett** [parcel-bundler/parcel#7890]
# 🐛 bug report

When importing an SVG image, the final build file contains meta.import.url. This happens only after running build. It doesn't happen when running dev. It causes my Firefox extension to fail validation -- preventing me from being able to submit an update that was built with the latest Parcel.

## 🔦 Context

I develop a Firefox extension, and after updating Parcel to the latest version -- I was previously using a nightly from last year -- my extension is now failing validation due to an error. `JavaScript syntax error (Parsing as script error: Cannot use 'import.meta' outside a module at line: 1 and column: 174752).`

The error is caused by the following code in the script after running build.
`i = new URL(t("27Lyk").resolve("3iRFs"), import.meta.url).toString();`

## 💻 Code Sample

A repo that can be used for reproducing this issue is below.
https://github.com/lucaseverett/parcel-svg-issue

The final build file is also in the repo.
https://github.com/lucaseverett/parcel-svg-issue/blob/master/dist/index.bce512b5.js

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 2.4.1
| Node             | 17.5.0
| npm/Yarn         |  npm 8.4.1
| Operating System | macOS

**Comment by lucaseverett** [comment]
I'm closing this as it's actually an issue with the Firefox linter. I'm not sure how to resolve yet, but I'll add a solution here when I find one in case anyone else encounters this.

**Comment by lucaseverett** [comment]
This issue was caused by the Firefox addons linter not scanning the script as a module.

The workaround was to manually change the filename from .js to .mjs. After changing to .mjs the scanner no longer had an issue with the script, and I was able to submit the update to my extension.

This was only an issue for the Firefox scanner. Chrome didn't find any issues with the .js file.

https://github.com/mozilla/addons-linter/issues/3639
